pkg/config/testdata/containers_default.conf

```diff
@@ -152,6 +152,12 @@ tmp_dir = "/run/libpod"
 
 machine_enabled = true
 
+# MachineImage is the image path or url used when creating a podman-machine VM
+machine_image = "/test/path"
```

Comment:
This path does not help me, and should probably be ""?

Reply:
This is a test containers.conf